storage/metrics_storage.go

```diff
@@ -160,14 +171,116 @@ func (ms *MetricsStorageManager) consumeMetrics() {
 	}
 }
 
-func getHawkularMetricsClient(conf *config.Config) (*hmetrics.Client, error) {
+func processHawkularMetricsConfig(conf *config.Config) (config.Hawkular_Server, error) {
+	client, clientErr := k8s.GetKubernetesClient(conf);
```

Comment:
assuming we must always be able to get a k8s client has now broken the ability to run HOSA outside of openshift.